static/api/v1/openapi.yaml

```diff
@@ -4793,6 +4795,28 @@ paths:
                       invocation_type: automatic
                       trusted_origins:
                         - 'http://localhost:8092'
+                Success Hosted Login:
+                  value:
+                    id: 73731b7f-eb76-4143-9b4b-81a720385f5a
+                    realm_id: caf2ff640497591a
+                    tenant_id: 00011f1183c67b69
+                    display_name: Pet Authenticator Configuration
+                    config:
+                      type: hosted_login
+                      authentication_methods:
+                        - subtle_crypto_passkey
+                      trusted_origins:
+                        - 'http://localhost:8092'
+                Success Hosted Web:
+                  value:
+                    id: 73731b7f-eb76-4143-9b4b-81a720385f5a
+                    realm_id: caf2ff640497591a
+                    tenant_id: 00011f1183c67b69
+                    display_name: Pet Authenticator Configuration
+                    config:
+                      type: hosted_web
+                      trusted_origins:
+                        - 'http://localhost:8092'
```

Comment:
I thought we deprecated Hosted Web. Did we miss a location?

Reply:
Oh, if they're all supposed to be removed, then yes. They are still a few artifacts under examples for CreateAuthenticatorConfig and UpdateAuthenticatorConfig. And the resource for the hosted login config is still called "Hosted Web Authenticator".

Reply:
@ismailmustafa, can you advise here? We probably don't want any references to "hosted_web" anymore, correct?